SLHCUpgradeSimulations/L1TrackTrigger/plugins/L1TkMuonFromExtendedProducer.cc

```diff
@@ -91,32 +92,66 @@ class L1TkMuonFromExtendedProducer : public edm::EDProducer {
   bool correctGMTPropForTkZ_;
 
   bool use5ParameterFit_;
-} ;
-
+};
+
+struct L1TkMuonCand {
+  float eta;
+  float phi;
+  float pt;
+  float feta;
+  float bx;
+  float sigmaEta;
+  float sigmaPhi;
+  unsigned int quality;
+  bool gmtCand;
+  bool dtCand;
+  bool cscCand;
+  bool rpcCand;
+};  
+
+// maximum Muons number
+const int MuonsMax = 10;
+
+struct L1TkCand {
+  float pt;
+  int   nPars;
+  int   q;
+  float z;
+  float chi2;
+  float eta;
+  float phi;
+  int   nstubs;
+  int   idx;
+  float matchPt;
+  float matchEta;
+  float matchPhi;
+  float matchSigmaEta;
+  float matchSigmaPhi;
+};  
+  
+// maximum Tracks number
+const int TrackMax = 300;
 
```

Comment:
we still need a struct for the output objects, L1TkMuon or L1TkMuonCand, it should contain all data necessary to fill L1TkMuonParticle